site/content/docs/09-building.md

```diff
@@ -12,4 +12,16 @@ The output is a Node app that you can run from the project root:
 
 ```bash
 node __sapper__/build
-```
\ No newline at end of file
+```
+
+### Browser support
+
+By default, Sapper builds your site only for the latest versions of modern evergreen browsers.
+
+For older browsers, like Internet Explorer, you will need to use the `--legacy` flag:
```

Comment:
I think we should be a bit more explicit about the fact that this will build *two* bundles and serve up the correct one at runtime. Otherwise people might think you need to build the app twice, one with the flag and one without, which won't work at all.

Reply:
Thinking about it we should also note that this only works for `Rollup` builds.

Reply:
That's an excellent call out I was not aware of! Is there anyway to support IE in Webpack?

Reply:
No, I think it was planned but other things took precidence.

To be honest I don't know how long we plan to support webpack for, it's kind-of a legacy decision which is no longer necessary.